clients/src/main/java/org/apache/kafka/common/requests/SaslHandshakeRequest.java

```diff
@@ -35,8 +38,13 @@
  * making it easy to distinguish from a GSSAPI packet.
  */
 public class SaslHandshakeRequest extends AbstractRequest {
+    private static final String MECHANISM_KEY_NAME = "mechanism";
 
-    public static final String MECHANISM_KEY_NAME = "mechanism";
+    private static final Schema SASL_HANDSHAKE_REQUEST_V0 = new Schema(
```

Comment:
Maybe later, but it seems like the names of these fields in every request/response class should basically be `SCHEMA_V0` or something.